app/src/main/java/com/example/lhi06/residue/AlbumReview.java

```diff
@@ -0,0 +1,44 @@
+package com.example.lhi06.residue;
+
+import android.support.annotation.NonNull;
+
+/**
+ * Created by lhi06 on 20/07/2017.
+ */
+
+public class AlbumReview {
+
+    private Album album;
+    private int rating;
+    private String review = "";
+
+    public AlbumReview(@NonNull final Album thisAlbum, final int rating, @NonNull final String review) {
+        this.album = thisAlbum;
+        this.rating = rating;
+        this.review = review;
+    }
+
+    public Album getThisAlbum() {
+        return album;
+    }
+
+    public void setThisAlbum(final Album thisAlbum) {
```

Comment:
i might as well delete the whole file if im doing that as i'm not using it at all until the next stage of the ticket (it's a big ticket so it wasnt worth me doing it all then pushing it) so are you sure i should delete the setters for now?